dnaseq_slurm_api.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# assumes crg2 is installed in home directory of user
+# in production, crg2 is located in /home/ccmmarvin
+# ANALYSIS_ID, FAMILY, and DATA are provided by STAGER
+
+set -euo pipefail
+
+SF=~/crg2/Snakefile; 
+CP="/home/slurm/conda_envs/crg2-conda";
+SLURM=~/crg2/slurm_profile;
+ANALYSIS_ID="$1"
+FAMILY="$2"
+DATA="$3"
+#assume a python dictionary is passed; parse this so it is compatible with exome_setup_stager.py
+DATA=`echo $DATA | sed 's/{//g' | sed 's/}//g' | sed 's/],/] /g'`
```

Comment:
This isn't efficient; I can simply pass in $DATA so it's the expected format. Since below the argument is passed as `-d $DATA`, the shell will automatically expand it so the whitespace in it will delimit the parameters passed to `exome_setup.py`. However, which file do we attach when there are many? I see that exomes can have four of varying types. And is this the full path or just the MinIO path?

Reply:
So here I'm assuming that you'll have a dictionary with {participant_codename: [linked files]}, where linked files includes all files associated with a dataset; the exome_setup_stager.py script will parse the [list](https://github.com/ccmbioinfo/crg2/pull/136/files#diff-ca4ef0fd17a8ca3b1e5d4555f5135c74dcf472cde9d185a39e80041a46e14e45R81) and choose the appropriate file. Not sure how better to pass the arguments in a 'linked' way.

Just the MinIO path is needed (I create the full path [here](https://github.com/ccmbioinfo/crg2/pull/136/files#diff-ca4ef0fd17a8ca3b1e5d4555f5135c74dcf472cde9d185a39e80041a46e14e45R88)).